src/server/config/express.js

```diff
@@ -55,6 +56,14 @@ export default (server) => {
     proxyOneQuran.web(req, res);
   });
 
+  server.use('/apple-app-site-association', (req, res) => {
```

Comment:
I want to say you can use express' static file helper with the route and add content type. Not certain but this works